Cargo.toml

```diff
@@ -48,6 +48,7 @@ directories = "5"
 displaydoc = "0.2"
 ethereum-types = "0.15"
 ez-jsonrpc-types = "0.3"
+fancy-duration = "0.9"
```

Comment:
It seems like a pretty niche dependency without many users - do you think it makes sense depending on it or can we avoid? https://crates.io/crates/fancy-duration/reverse_dependencies

Reply:
Let me see if I can find a better replacement.  I first spotted https://crates.io/crates/human-duration but found it's not been updated for 3 years

Reply:
@LesnyRumcajs `fancy-duration` is the best I can find under https://crates.io/search?q=duration

`chrono::Duration` displays 15s as `PT15s` which follows https://en.wikipedia.org/wiki/ISO_8601#Durations but is not super readable for non-tech users

Reply:
What about `humantime`? It's a bit of an abandonware, but we already have it in our dependencies.